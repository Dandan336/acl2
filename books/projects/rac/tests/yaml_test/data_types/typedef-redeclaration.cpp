// RAC begin

typedef int MyType;
typedef int MyType;

// RAC end
