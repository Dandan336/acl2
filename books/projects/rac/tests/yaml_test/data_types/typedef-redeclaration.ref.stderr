typedef-redeclaration.cpp:4: syntax error, unexpected TYPEID, expecting ID
